src/ServiceProvider.php

```diff
@@ -77,6 +77,15 @@ public function register( PimpleContainer $container ) {
 			);
 		};
 
+		$container['authentication.disable'] = function( $container ) {
+			$server_count = \iterator_count( $container['authentication.servers'] );
+			if ( 0 >= $server_count ) {
```

Comment:
I would suggest just returning the count value rather than an if else statement. Example: `return 0 >= $server_count;